var/spack/repos/builtin/packages/mesa/package.py

```diff
@@ -123,7 +127,8 @@ def meson_args(self):
 
         opt_enable = lambda c, o: '-D%s=%sabled' % (o, 'en' if c else 'dis')
         opt_bool = lambda c, o: '-D%s=%s' % (o, str(c).lower())
-        if spec.target.family == 'arm' or spec.target.family == 'aarch64':
```

Comment:
@mwkrentel this line made me exclude arm and aarch64. Unfortunately I don't know the exact reasoning behind this line.